scipy/signal/signaltools.py

```diff
@@ -416,6 +416,18 @@ def convolve(in1, in2, mode='full'):
         ``same``
            The output is the same size as `in1`, centered
            with respect to the 'full' output.
+    method : str {'auto', 'fft', 'direct'}, optional
+        A string indicating what method used to perform the convolution.
+
+        ``auto``
+           A rough estimate of which method below is faster is performed and
+           that method is chosen.
+        ``fft``
+           The Fourier Transform method is used to perform the convolution.
+           Equivalent to `fftconvolve`.
+        ``direct``
+           The convolution is determined directly from sums, the definition of
+           convolution.
```

Comment:
I suspect this will render more nicely if you indent these lines, but I'm not sure. Have you looked at how the doc builds like this?